pkg/hubble/parser/threefour/parser_test.go

```diff
@@ -802,6 +802,17 @@ func TestDecodeIsReply(t *testing.T) {
 	assert.Nil(t, f.GetIsReply())
 	assert.Equal(t, false, f.GetReply())
 
+	// TRACE_TO_STACK Encrypted Overlay
+	tn = monitor.TraceNotifyV0{
+		Type:     byte(monitorAPI.MessageTypeTrace),
+		Source:   localEP,
```

Comment:
I think this needs to be
```suggestion
		Source:   hostEP,
```
Since the trace is emitted by `bpf_host.c`

Reply:
@kaworu sounds good to me, but should the test have failed? It passes like this.

Reply:
No I think it's expect too pass either way, it's was more like a cleanup request :pray: